01.fizzbuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,12 @@
+# 1から20までの数をプリントするプログラムを書け。ただし3の倍数のときは数の代わりに｢Fizz｣と、5の倍数のときは｢Buzz｣とプリントし、3と5両方の倍数の場合には｢FizzBuzz｣とプリントすること。
+1.upto(20) do |num|
+  if num % 3 ==0 && num % 5 == 0
```

Comment:
他のコードと一貫性を持たせるため

```suggestion
  if num % 3 == 0 && num % 5 == 0
```

のようにスペースを空けましょう。